### PR TITLE
Kmk mt metric

### PR DESCRIPTION
**Pull request by kishorikonwar**
### Purpose
To add mitochondrial gene counts to the cell metrics

### Changes
 Changed the CalcualteCellMetrics input to add the reference gtf files and the docker for sctools in the same task

### CODE REVIEW

File: library/tasks/SequenceDataWithMoleculeTagMetrics.wdl [hunk 1]
```diff
@@ -47,9 +47,10 @@ task CalculateGeneMetrics {
 task CalculateCellMetrics {
   input {
     File bam_input
+    File original_gtf
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.7"
+    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:kmk-mt-counts"
```
Comment (jessicaway) [inline library/tasks/SequenceDataWithMoleculeTagMetrics.wdl:53]: can you rename the docker with a version number?

**Comment by kishorikonwar** [comment]
retest

File: library/tasks/SequenceDataWithMoleculeTagMetrics.wdl (hunk 1 above)
Comment (kishorikonwar) [reply library/tasks/SequenceDataWithMoleculeTagMetrics.wdl]: Yes, the version number is changed.

File: library/tasks/ZarrUtils.wdl [hunk 2]
```diff
@@ -93,12 +93,12 @@ task OptimusZarrConversion {
   command {
     set -euo pipefail
 
-
-    if ~{counting_mode} == "sc_rna"
-    then
+    if [ "${counting_mode}" == "sc_rna" ]; then
         EXPRESSION_DATA_TYPE_PARAM="exonic" 
+        ADD_EMPTYDROPS_DATA="yes"
```
Comment (jessicaway) [inline library/tasks/ZarrUtils.wdl:98]: Can we use boolean values instead of strings here?

Comment (kishorikonwar) [reply library/tasks/ZarrUtils.wdl:98]: We could but the python script, for which this variable is prepared, requires strings as argument options.

**Pull request merged**